src/java.desktop/share/classes/javax/swing/plaf/basic/BasicTextUI.java

```diff
@@ -647,6 +647,21 @@ ActionMap createActionMap() {
                 TransferHandler.getCopyAction());
         map.put(TransferHandler.getPasteAction().getValue(Action.NAME),
                 TransferHandler.getPasteAction());
+
+        if (getComponent() instanceof JPasswordField) {
```

Comment:
Since the fix has been made in Basic, we should not restrict the test to Aqua L&F alone. It should be tested for all L&F. (For fix and for regression)